### DISCUSSION TITLE
Filter option for the newsfeed by one or more of my selected tags

### DISCUSSION THREAD

**Discussion by maxbleh** [dailydotdev/daily#867] [Feature Requests]
hi daily.dev Team,

first of all, i love daily.dev, you did an awesome Job!  

From time to time i think it would be nice if i can select one or more of my defined tags and show only content for those tags.

For example:
My feed ist Configured to show "Flutter", "C#", "Dart", "React", "React-Native". My feed shows content for all of those tags. It would be awesome if i could say "Show only Flutter" 

Maybe it would be possible to Show all my selected Tags as Chips Toggle above the News Feed to switch tags of and on 

I hope I didnt formulated my idea to confusing :)

**Comment by ghost** [comment]
This is the feature I look forward to the most.

**Comment by idoshamun** [comment]
Hi @blehgmbh,

Thanks a lot for the idea. Makes total sense.

**Comment by maxbleh** [comment]
Awesom! Im looking forword to it

**Comment by dongruixiao** [comment]
I have the same idea. The current feed stream feels overwhelming to me, and I might prefer to focus on viewing specific tags of content.
